packages/core/test/interpreter.test.ts

```diff
@@ -184,14 +184,14 @@ describe('interpreter', () => {
         }
       });
 
-      const s = interpret(machine);
+      const bState = machine.initialState;
+
+      expect(bState.value).toEqual('b');
 
-      // would have deferred the entry action
-      s.getSnapshot();
+      // Ensure that there are no actions attached to the state
+      bState.actions.length = 0;
```

Comment:
Couldn't this be just replaced with `machine.resolveStateValue('b')`? It seems more straightforward than resetting actions on an object that should be treated as immutable